packages/react-reconciler/reflection.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ */
+
+'use strict';
+
+export * from './src/ReactFiberTreeReflection';
```

Comment:
Can we use a `require` here for consistency with other top level entry points?
i.e.

```js
module.exports = require('./src/ReactFiberTreeReflection');
```

We can eventually change this but I'd like all entry points to look the same.

Reply:
Nvm, I see why this wouldn't work now.